src/components/ClassSearch.vue

```diff
@@ -164,6 +164,7 @@ export default {
           'level': getRegexFuncs(this.chosenFilters.levelInput),
           'total_units': getRegexFuncs(this.chosenFilters.unitInput)
         };
+        // TODO: remove?
```

Comment:
@georgiashay I think you kept this commented out stuff around for a reason earlier, but that's probably unnecessary now right?

Reply:
Left it for discovering all values of a given attribute on a subject.  Could still be useful but it'd be better to just copy paste somewhere and use it when needed than keep it commented out in the codebase.